app/Http/Controllers/API/AmasController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Ama;
+
+class AmasController
+{
+    public function getAma($id)
+    {
+        $ama = Ama::where('id', '=', $id)->first();
```

Comment:
You can also do `Ama::find($id)`, that would return the Ama record by id directly.